ci-operator/step-registry/redhat-chaos/pvc-scenarios/redhat-chaos-pvc-scenarios-chain.yaml

```diff
@@ -3,6 +3,5 @@ chain:
   as: redhat-chaos-pvc-scenarios
   steps:
   - ref: redhat-chaos-pvc-scenarios-prometheus-pod
-  - ref: redhat-chaos-pvc-scenarios-new-pod
```

Comment:
is there a reason we're removing this ref?

Reply:
redhat-chaos-pvc-scenarios-new-pod is more like a test scenario as it creates a test pod with a PVC and fills it up: https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/pvc-scenarios/new-pod/redhat-chaos-pvc-scenarios-new-pod-ref.yaml#L20. We should probably remove it. Thoughts?

Reply:
Removed the scenario @paigerube14. PTAL when you get time please.